### PR TITLE
Adds Syndie MMI as Standalone Option for Roboticists and RD

### PR DESCRIPTION
**Pull request by DatBoiTim**
## About The Pull Request
This adds the Syndicate MMI as a Standalone item for Roboticists and Research Directors to buy from the Uplink.

Adding this can allow for an item found in a criminally underused uplink set as an item. Allowing for sneakier use of this item. Additionally, as a roboticist, the medical duffle would be redundant, as you already have most of the items within.

Slight increase to cost of the syndie surgery bag to account for the added value of the MMI.

## Why It's Good For The Game
Adds more varied options for some roles without undermining the alternative completely. Allows for additional strategies and loadouts to be used. Also allows for said roles to cut down on redundancy should they desire the Syndicate MMI but not want the full surgical kit. As such allows for said additional tool to be used more as redundancy is no longer a factor.

## Changelog
:cl: DatBoiTim
tweak: Syndicate MMI is a standalone uplink item. Restricted to Roboticist and RD.
tweak: Increased Syndicate Surgery Duffle Bag cost to 3 TC
/:cl:

### CODE REVIEW

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1907,6 +1907,13 @@ datum/uplink_item/stealthy_tools/taeclowndo_shoes
 	restricted_roles = list("Medical Doctor", "Chief Medical Officer", "Roboticist", "Paramedic")
 	cost = 3
 
+/datum/uplink_item/rolerestricted/Syndicate_MMI
+	name = "Syndicate MMI"
+	desc = "An MMI which autmatically applies the Syndimov laws to any borg it is placed in. Great for adding known allies to assist you with a little more stealth than a fully emagged borg."
+	item = /obj/item/mmi/syndie
+	restricted_roles = list("Roboticist", "Research Director")
+	cost = 1
```
Comment (Archanial) [inline code/modules/uplink/uplink_items.dm]: 1 is too low for effectively emag effect. I would suggest 2.

Comment (DatBoiTim) [reply code/modules/uplink/uplink_items.dm]: Idk tbh. It's only laws. In addition the kit which includes tools plus restraints on top of the MMI is 2 TC.

I could alternatively push the standalone to 2 and kit to 3

Comment (DatBoiTim) [reply code/modules/uplink/uplink_items.dm]: Resolved by: 34eea88
Updating Changelog

**Comment by Singularity69** [comment]
Uhhhh whats with the sec CQC/martial art thing?

**Comment by ghost** [comment]
Uhhhh whats with the sec CQC/martial art thing?

**Comment by DatBoiTim** [comment]
> Uhhhh whats with the sec CQC/martial art thing?

Son of bitch I didn't stash those changes and they got brought over to my SyndieMMI branch.

**That is a part of a whole other project I am working on, and is not meant to be in this PR**

**Comment by DatBoiTim** [comment]
> Uhhhh whats with the sec CQC/martial art thing?

Fixed. 0e4ec27 is a 34eea889d4 redux, with changes stashed/committed to correct branches.

**Pull request merged**